monero-wallet-gui.pro

```diff
@@ -274,10 +275,13 @@ win32 {
     }
 }
 
-linux {
+linux:!android {
     deploy.commands += $$escape_expand(\n\t) $$PWD/linuxdeploy_helper.sh $$DESTDIR $$TARGET
 }
 
+android{
+    deploy.commands += make install INSTALL_ROOT=$$DESTDIR && androiddeployqt --input android-libmonero-wallet-gui.so-deployment-settings.json --output $$DESTDIR --deployment bundled --android-platform android-21 --jdk /usr/lib/jvm/java-8-openjdk-amd64 -qmldir=$$PWD
```

Comment:
Did you forget to commit android-libmonero-wallet-gui.so-deployment-settings.json?